source/synthSettingsRing.py

```diff
@@ -13,17 +13,42 @@ def __init__(self,synth,setting,min=0,max=100):
 		self.min = setting.minVal if isinstance(setting, NumericDriverSetting) else min
 		self.max = setting.maxVal if isinstance(setting, NumericDriverSetting) else max
 		self.step = setting.normalStep if isinstance(setting, NumericDriverSetting) else 1
+		self.largeStep = self.setting.largeStep if isinstance(setting, NumericDriverSetting) else 10
+
+	def first(self) -> Any:

```

Comment:
`_getReportValue` returns a str. Please update these function type hints to specify the actual return type

```suggestion
	def first(self) -> str:
```